### ISSUE TITLE
 Which graphics card is recommended

### ISSUE THREAD

**Issue by cavencj** [CesiumGS/cesium#9202]
Now there are two graphics cards NVIDIA Quadro RTX8000 48G and (ZOTAC)RTX3080
Which one can render Cesium better，

**Comment by midnight-dev** [comment]
They both should be able to render Cesium with globe, atmosphere, imagery and terrain at any zoom level at the max frame rates that your browser will allow. Likewise, they're both overkill for rendering large models. You'd only need accelerators that powerful if you want to display a high number of 3D tilesets with a large amount of 3D tiles and/or very detailed imagery/textures.

For comparison, I have a GTX 1060 6GB driving 2x 4K displays, and Firefox can reliably render a photogrammetry tileset within a Cesium viewer, and it all runs at 60 FPS if I don't intentionally throttle the frames. Now, there are still a few hiccups when everything has to first load in, but that's going to happen anyway due to the immense size of Cesium. It's a lot of code to parse and execute, and then terrain & imagery have to be downloaded and applied to a globe. The same goes for any models or custom imagery. That hitching at initial load can only be minimized, not bypassed. The CPU, memory, and internet connection largely determine how noticeable that load will be. After things initialize, you'll get smooth frame rates from either of those GPUs.

While future specs for WebAssembly and WebGPU might be able to take advantage of accelerating double precision operations, I don't believe WebGL rendering would benefit from the higher performing half & double precision operations on Quadros. A Quadro is very nice for simulations, ML, and anything that needs a _**large**_ frame buffer, but a flagship GeForce card will clock higher and perform better in consumer rendering tasks (e.g. gaming, accelerating web browsers). The RTX 8000 is actually based on last generation's Turing chip anyway - it's closer to a 2080 than a 3080. The Ampere version isn't quite out yet, but its name will be A6000 (no 8000 model). Therefore... 

## TL;DR: 

If you don't need the extra features of Quadro cards for something else, get the 3080. It's newer, boosts higher, and is much cheaper.

**Comment by cavencj** [comment]
@midnight-dev  thanks，i get it

**Comment by dzungpng** [comment]
Hi there,

For future questions related to Cesium, please post them on the Cesium community forum: https://community.cesium.com/!
